.circleci/config.yml

```diff
@@ -78,6 +77,19 @@ mysql_service: &mysql_service
 version: 2.0
 workflows:
   version: 2
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 6 * * *" # Daily at 6:00AM
```

Comment:
qq: do we know what timezone?

Reply:
It's UTC. I didn't think about this until you asked 🙃 

which makes it 10:00PM PT. I think that's reasonable since it's outside of normal work hours.